.eslintrc

```diff
@@ -63,35 +56,75 @@ rules:
   ## require parens for Constructor
   new-parens: 2
   ## max 80 length
-  max-len:
-    - 2
-    - 80
-    - 2
+  max-len: [2, 80, 2]
 
   # Strict Mode
   # list: https://github.com/eslint/eslint/tree/master/docs/rules#strict-mode
   ## 'use strict' on top
-  strict:
-    - 2
-    - "global"
+  strict: [2, "global"]
+
+  # Variables
+  # list: https://github.com/eslint/eslint/tree/master/docs/rules#variables
+  ## disallow use of undefined variables (globals)
+  no-undef: 2
 
 # Global scoped method and vars
 globals:
-  DTRACE_HTTP_CLIENT_REQUEST: true
-  LTTNG_HTTP_CLIENT_REQUEST: true
-  COUNTER_HTTP_CLIENT_REQUEST: true
-  DTRACE_HTTP_CLIENT_RESPONSE: true
-  LTTNG_HTTP_CLIENT_RESPONSE: true
-  COUNTER_HTTP_CLIENT_RESPONSE: true
-  DTRACE_HTTP_SERVER_REQUEST: true
-  LTTNG_HTTP_SERVER_REQUEST: true
-  COUNTER_HTTP_SERVER_REQUEST: true
-  DTRACE_HTTP_SERVER_RESPONSE: true
-  LTTNG_HTTP_SERVER_RESPONSE: true
-  COUNTER_HTTP_SERVER_RESPONSE: true
-  DTRACE_NET_STREAM_END: true
-  LTTNG_NET_STREAM_END: true
-  COUNTER_NET_SERVER_CONNECTION_CLOSE: true
-  DTRACE_NET_SERVER_CONNECTION: true
-  LTTNG_NET_SERVER_CONNECTION: true
-  COUNTER_NET_SERVER_CONNECTION: true
+  # Node
+  __filename        : false
+  __dirname         : false
+  global            : false
+  GLOBAL            : false
+  root              : false
```

Comment:
`GLOBAL` and `root` are going to be deprecated (#1838). Are they still being used in the code base ? If not they do not need to be here.